knowledge-base/chart-draw-horizontal-vertical-lines.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Charts - Draw Horizontal and Vertical Lines
+description: How to draw horizontal lines in Charts. How to draw vertical lines in Charts.
+type: how-to
+page_title: How to Draw Horizontal and Vertical Lines in Charts
+slug: chart-kb-draw-horizontal-and-vertical-lines
+position: 
+tags: telerik, blazor, chart, draw, horizontal, vertical, lines
+ticketid: 1511761, 1569732
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Charts for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to draw horizontal lines in Telerik Blazor Charts?
+* How to draw vertical lines in Telerik Blazor Charts?
+* Is it possible to draw vertical or horizontal lines on the Telerik Blazor Chart?
```

Comment:
These three questions use the same keywords. You can enhance them and add more keywords from the tickets:

markers, limits, thresholds, pins